volapi/chat.py

```diff
@@ -54,9 +58,10 @@ class ChatMessage(str):
 
     # pylint: disable=no-member
 
-    def __new__(cls, room, conn, nick, msg, role=Roles.WHITE, options=None, **kw):
-        if role not in Roles:
-            raise ValueError("Invalid role")
+    def __new__(cls, room, conn, nick, msg, role=[Roles.WHITE], options=None, **kw):
```

Comment:
Also since user can have multiple roles, I think its sensible to change the name of this property in `__new__` constructor from `role` to `roles`
```suggestion
    def __new__(cls, room, conn, nick, msg, roles={Roles.WHITE}, options=None, **kw):
```

Reply:
Did This ✔️